components/inbound-endpoints/org.wso2.carbon.inbound.endpoint/src/main/java/org/wso2/carbon/inbound/endpoint/protocol/websocket/InboundWebsocketConstants.java

```diff
@@ -30,6 +30,8 @@
     public static final String INBOUND_SSL_TRUST_STORE_FILE = "wss.ssl.trust.store.file";
     public static final String INBOUND_SSL_TRUST_STORE_PASS = "wss.ssl.trust.store.pass";
     public static final String INBOUND_SSL_CERT_PASS = "wss.ssl.cert.pass";
+    public static final String SSL_PROTOCOLS = "wss.ssl.protocols";
+    public static final String CIPHER_SUITES = "wss.ssl.cipher.suites";
```

Comment:
Can we change the parameter name as PreferredCiphers since other transport use PreferredCiphers as the parameter name

Reply:
Since we have already used this parameter name for ESB 5.0.0, it would be inconsistent if we change the name here. Shall we move with this ?